en/includes/guides/authentication/mfa/add-smsotp-login.md

```diff
@@ -76,116 +77,6 @@ To update the default SMS OTP settings:
-    - Fill in the required fields and create the account.
-    - Login to the Vonage dashboard and copy the API Key and API Secret.
-    - Go to **SMS Provider** section in the {{ product_name }} Console and click the **Vonage** tab and fill the required fields.
-
-    <table>
-      <tr>
-        <th>Name</th>
-        <th>Description</th>
-        <th>Example</th>
-      </tr>
-      <tr>
-        <td>Vonage API Key</td>
-        <td>Use the API Key from the previous step.</td>
-        <td><code>YourAPIKey</code></td>
-      </tr>
-      <tr>
-        <td>Vonage API Secret</td>
-        <td>Use the API Secret from the previous step.</td>
-        <td><code>YourAPISecret</code></td>
-      </tr>
-      <tr>
-        <td>Sender</td>
-        <td>The number that the receiver will see when you send an SMS.</td>
-        <td><code>+1234567890</code></td>
-      </tr>
-    </table>
-
-??? note "Configuring a Custom Provider"
-    If you are not using either Twilio or Vonage as the SMS provider, you can configure a custom SMS provider. Custom SMS provider configuration will pass the SMS data to the given URL as an HTTP request.
-
-    To configure a custom SMS provider, in the  **SMS Provider** section click the **Custom** tab and fill the required fields.
-
-    <table>
-      <tr>
-        <th>Name</th>
-        <th>Description</th>
-        <th>Example</th>
-      </tr>
-      <tr>
-        <td>SMS Provider URL</td>
-        <td>URL of the SMS gateway where the payload should be published.</td>
-        <td><code>https://api.example.com/api/v1</code></td>
-      </tr>
-      <tr>
-        <td>Content Type</td>
-        <td>Content type of the payload. Possible values are <code>JSON</code> or <code>FORM</code> (Optional).</td>
-        <td><code>JSON</code></td>
-      </tr>
-      <tr>
-        <td>HTTP Method</td>
-        <td>HTTP method that should be used when publishing the payload to the provider URL. Possible values: <code>PUT</code>, <code>POST</code> (Optional). </td>
-        <td><code>POST</code></td>
-      </tr>
-      <tr>
-        <td>Payload Template</td>
-        <td>How the payload template should be. </br>Placeholders: </br><code>\{\{body\}\}</code> - Generated body of the SMS. (Example - This can be the OTP). </br><code>\{\{mobile\}\}</code> - Number that this sms should be sent to.</td>
-        <td>Example JSON payload template: </br><code>{“content”: \{\{body\}\},“to”: \{\{mobile\}\}}}</code></br></br>(<code>\{\{mobile\}\}</code> and <code>\{\{body\}\}</code> will be replaced with the corresponding values at the runtime.)</td>
-      </tr>
-      <tr>
-        <td>Headers</td>
-        <td>Custom static headers need to be passed. If multiple headers need to be passed, they should be comma separated. (Optional)</td>
-        <td><code>authorisation: qwer1234asdfzxcv, x-csrf: true, x-abc: some-value</code></td>
-      </tr>
-    </table>
-
 ## Enable SMS OTP for an app
```

Comment:
Here also, if its not already indicated, let's add a note with a link to sms provider setup doc.

Reply:
This is already added in https://github.com/wso2/docs-is/pull/5308/files#diff-76d4133ee02debb7d38cc552543f67987eb0fc3f274cd7bec45ef3182f269a90R13 prerequisites.